src/components/Auth/AuthContainer.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+
+import Auth from './Auth';
+import { registerUser } from '../../actions/auth';
+
+
+const mapStateToProps = state => ({
+  register: state.register,
```

Comment:
You can also use lodash's `get` to extract keys. It allows for setting easy defaults.